libs/payments/iap/src/lib/google/google-iap.service.ts

```diff
@@ -25,15 +21,8 @@ export class GoogleIapService {
 
   async getActiveSubscriptions(
     uid: string
-  ): Promise<AppendedPlayStoreSubscriptionPurchase[]> {
-    const purchases = await this.purchaseManager.getForUser(uid);
-
-    // Below to be completed as part of FXA-7839
-    //return this.stripeHelper.addPriceInfoToIapPurchases(
-    //  purchases,
-    //  MozillaSubscriptionTypes.IAP_GOOGLE
-    //);
-    return purchases as AppendedPlayStoreSubscriptionPurchase[];
+  ): Promise<PlayStoreSubscriptionPurchase[]> {
+    return this.purchaseManager.getForUser(uid);
```

Comment:
It'd be great if we used the manager directly from the subscription management service so it's service->manager rather than service->service->manager